srcpkgs/avidemux/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'avidemux'
 pkgname=avidemux
 version=2.7.4
-revision=5
+revision=6
```

Comment:
Any reason why it has to be rebuild?

Reply:
Strictly speaking no. I can remove if that is preferable. I erred on the side of caution, I haven't noticed any problems with it though